modelforge/train/training.py

```diff
@@ -359,6 +375,7 @@ def _update_metrics(
         """
 
         for prop, metric_collection in metrics.items():
+            prop = _exchange_per_atom_energy_to_per_molecule_energy(prop)
```

Comment:
it would be good to add a comment that this only modifies `per_atom_energy` and all other properties are left unchanged.